File: examples/hello.py
def diff(first, second):
  intersection = [k for k in first if k in second]
  addition = [k for k in second if k not in first]
  deletion = [k for k in first if k not in second]

  return intersection, addition, deletion
    

first = {'lorem': 1, 'amet': 5}
second = {'ipsum': 2, 'dolor': 3, 'sit': 4}
    
result = diff(first, second)

_, added, removed = result

print 'diffed dicts'
print first, second

print 'added keys:'
print added

print 'removed keys:'
print removed

File: docker/hello.js
const contains = (key, list) => {
    return list.indexOf(key) > -1;
}

const diff = (first, second) => {
	let firstKeys = Object.keys(first);
	let secondKeys = Object.keys(second);

	let addition = secondKeys.filter(key => !contains(key, firstKeys))
	let deletion = firstKeys.filter(key => !contains(key, secondKeys))

	return {addition, deletion}
}

let first = {'lorem': 1, 'ipsum': 2};
let second = {'dolor': 3, 'sit': 4, 'lorem': 1};

console.log('first object')
console.log(first)

console.log('second object')
console.log(second)

console.log('diff')
console.log(diff(first, second))
